conda-recipes/ray-feedstock/recipe/conda_build_config.yaml

```diff
@@ -0,0 +1,2 @@
+python:
+  - 3.6
```

Comment:
Maybe more for @jayfurmanek .... All of the conda-forge packages seem to be single-python verion (the build string is just `py_0`, rather than, say `py36_0`). That leaves out folks that might want to use this with WML CE 1.6.2 w/ Python 3.7.

Reply:
I removed the conda-forge channel (cf other comment). Installing the package failed with python 3.7, some dependencies were not resolved. Maybe now without conda-forge and the new packages/versions from WMLCE 1.6.2 it works.
I don't know how to change the build string (py_0), feel free to modify the recipe if needed